packages/zoltan2/test/unit/models/GraphModel.cpp

```diff
@@ -127,6 +127,63 @@ void printGraph(zlno_t nrows, const zgno_t *v,
   comm->barrier();
 }
 
+/////////////////////////////////////////////////////////////////////////////
+
+template <typename MatrixOrGraph>
+void computeNumDiags(
+    RCP<const MatrixOrGraph> &M,
```

Comment:
M is an input argument, not an output argument, so the RCP should be passed by value or const reference:
```
void computeNumDiags(
    const RCP<const MatrixOrGraph> &M,
    size_t &numLocalDiags,
    size_t &numGlobalDiags
);
```